deid/dicom/parser.py

```diff
@@ -301,7 +311,7 @@ def get_fields(self, expand_sequences=True):
                 dicom=self.dicom,
                 expand_sequences=expand_sequences,
                 seen=self.seen,
-                skip=self.skip,
+                skip=self.skip
```

Comment:
Did you run a recent version of black on this? Usually black adds a trailing comma.